bdb/file.c

```diff
@@ -4493,7 +4488,7 @@ int open_dbs(bdb_state_type *bdb_state, int iammaster, int upgrade, int create,
                     logmsg(LOGMSG_FATAL, "deadlock in open\n");
                     exit(1);
                 }
-                print(bdb_state, "open_dbs: cannot open %s: %d %s\n",
```

Comment:
I don't know what `print(bdb_state...` is used for. If it is still useful then I can add it back.